app/src/main/res/layout-land/hints_and_solution_fragment.xml

```diff
@@ -0,0 +1,71 @@
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <data>
+
+    <import type="android.view.View" />
+
+    <variable
+      name="viewModel"
+      type="org.oppia.app.player.state.hintsandsolution.HintsViewModel" />
+  </data>
+
+  <androidx.constraintlayout.widget.ConstraintLayout
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/white">
+
+    <com.google.android.material.appbar.AppBarLayout
+      android:id="@+id/hints_and_solution_app_bar_layout"
+      android:layout_width="0dp"
+      android:layout_height="wrap_content"
+      android:theme="@style/ThemeOverlay.AppCompat.Dark.ActionBar"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toTopOf="parent">
+
+      <androidx.appcompat.widget.Toolbar
```

Comment:
Please follow the attributes as per toolbar.xml layout-land.

Reply:
In one of the PR mentioned by @mschanteltc  toolbar textsize is same as portrait 20sp.

Reply:
I have updated status bar color and toolbar size.
![Screenshot_20200417-114235](https://user-images.githubusercontent.com/16301028/79537916-eac31e80-80a0-11ea-821f-3b068a7e1075.png)